global-institute-of-technology-and-business.csl

```diff
@@ -0,0 +1,550 @@
+      <term name="month-10" form="short">Okt</term>
+      <term name="month-11" form="short">Nov</term>
+      <term name="month-12" form="short">Des</term>
+      <term name="season-01">Semi</term>
+      <term name="season-02">Panas</term>
+      <term name="season-03">Gugur</term>
+      <term name="season-04">Dingin</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="translator editor" delimiter="; " suffix=".">
+      <name delimiter=", " delimiter-precedes-last="always" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
+      <label prefix=", "/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author" delimiter=", ">
+      <name delimiter-precedes-last="always" initialize-with="" name-as-sort-order="all" sort-separator=" "/>
+      <et-al font-style="normal"/>
+      <label form="long" prefix=", " strip-periods="true"/>
+      <substitute>
+        <names variable="translator editor"/>
+        <text variable="title"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" delimiter=", " initialize-with="." and="text"/>
+      <et-al font-style="normal"/>
+      <substitute>
+        <names variable="translator"/>
+        <names variable="editor"/>
+        <text variable="title" form="short" suffix="…"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="review">
+    <group delimiter=". ">
+      <text variable="reviewed-title"/>
+      <text variable="container-title"/>
+    </group>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="URL">
+        <group prefix=" " delimiter=" ">
+          <group prefix="[" suffix="]." delimiter=" ">
+            <text term="accessed"/>
+            <date variable="accessed" delimiter=" ">
+              <date-part name="year"/>
+              <date-part name="month" form="short"/>
+              <date-part name="day"/>
+            </date>
+          </group>
+          <text variable="URL" suffix="."/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <group delimiter=" ">
+      <choose>
+        <if type="book">
+          <text variable="title" font-style="italic"/>
+        </if>
+        <else>
+          <text variable="title"/>
+        </else>
+      </choose>
+      <choose>
+        <if variable="volume issue" match="any">
+          <text variable="note" prefix="[" suffix="]"/>
+        </if>
+        <else-if type="thesis">
+          <text variable="genre" prefix="[" suffix="]"/>
+        </else-if>
+      </choose>
+    </group>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=": ">
+      <text variable="publisher-place"/>
+      <text variable="publisher" suffix="."/>
+    </group>
+  </macro>
+  <macro name="pages">
+    <group delimiter="; ">
+      <group>
+        <label variable="page" form="short" suffix=" " plural="never"/>
```

Comment:
The suffix here should be better set on the group level as a delimiter. Please change and check the rest of your style for such instances.